pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["hatchling"]
```

Comment:
why do you use `hatchling` build system instead of `setuptools`. There are numerous Python build back-end with their own pros and cons. But as an example template, I would go for the Python official `setuptools`.

```toml
requires = [
    "setuptools",
    "wheel",
]
build-backend = "setuptools.build_meta"
```

Reply:
I picked this one becuase the python toturial also used hatchling